docker/synapse.Dockerfile

```diff
@@ -8,7 +8,7 @@ ENV DEBIAN_FRONTEND noninteractive
 
 RUN apt-get -qq update && apt-get -qq install -y \
         apt-utils ${PYTHON_VERSION} ${PYTHON_VERSION}-dev ${PYTHON_VERSION}-venv \
-        python3-pip eatmydata redis-server
+        python3-pip eatmydata redis-server cur
```

Comment:
is this meant to be `curl`?

Reply:
SIGH, yes thank you. My crankness continues